doc/source/train/user-guides/hyperparameter-optimization.rst

```diff
@@ -3,127 +3,168 @@
 Hyperparameter Tuning with Ray Tune
 ===================================
 
-Hyperparameter tuning with :ref:`Ray Tune <tune-main>` is natively supported with Ray Train.
+.. important::
+    This user guide shows how to integrate Ray Train and Ray Tune to tune over distributed hyperparameter runs
+    for the revamped Ray Train V2 available starting from Ray 2.43 by enabling the environment variable ``RAY_TRAIN_V2_ENABLED=1``.
+    **This user guide assumes that the environment variable has been enabled.**
 
+    Please see :ref:`here <train-tune-deprecation>` for information about the deprecation and migration.
 
-.. https://docs.google.com/drawings/d/1yMd12iMkyo6DGrFoET1TIlKfFnXX9dfh2u3GSdTz6W4/edit
 
-.. figure:: ../images/train-tuner.svg
+Ray Train can be used together with Ray Tune to do hyperparameter sweeps of distributed training runs.
+This is often useful when you want to do a small sweep over critical hyperparameters,
+before launching a run with the best performing hyperparameters on all available cluster resources for a long duration.
+
+Quickstart
+----------
+
+In the example below:
+
+* :class:`~ray.tune.Tuner` launches the tuning job, which runs a bunch of ``train_driver_fn`` trials with different hyperparameter configurations.
+* ``train_driver_fn``, which instantiates a ``TorchTrainer`` (or some other framework trainer), takes in a hyperparameter configuration and launches the distributed training job.
+* :class:`~ray.train.ScalingConfig` defines the number of training workers and resources per worker for a single Ray Train run.
+* ``train_fn_per_worker`` is the Python code that executes on each distributed training worker for a trial.
+
+.. literalinclude:: ../doc_code/train_tune_interop.py
+    :language: python
+    :start-after: __quickstart_start__
+    :end-before: __quickstart_end__
+
+
+Configuring Resources for Multiple Trials
+-----------------------------------------
+
+Ray Tune launches multiple trials which run a user-defined function in a remote Ray actor, where each trial gets a different sampled hyperparameter configuration.
+Typically, these Tune trials do work computation directly inside the Ray actor. For example, each trial could request 1 GPU and do some single-process model
+training within the remote actor itself. When using Ray Train inside Ray Tune functions, the Tune trial is actually not doing extensive computation inside this actor
+-- instead it just acts as a driver process to launch and monitor the Ray Train workers running elsewhere.
```

Comment:
When reading this section I expected to see some information about using `ScalingConfig` to specify resources.

Reply:
This would then transition nicely to the sections of setting max concurrency and tune function resources